packages/wrangler/src/d1/formatTimeAgo.ts

```diff
@@ -2,7 +2,9 @@ import TimeAgo from "javascript-time-ago";
 import en from "javascript-time-ago/locale/en";
 import prettyBytes from "pretty-bytes";
 
-TimeAgo.addDefaultLocale(en);
+TimeAgo.addLocale(en);
+TimeAgo.setDefaultLocale("en");
```

Comment:
Previously, `javascript-time-ago` complained about multiple calls of `addDefaultLocale()`

Reply:
Why is one `en` and the other `"en"`?

Reply:
One is a string, and the other is an object import from `TimeAgo`